.github/workflows/stale-issue-needs-info.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue is being marked as stale because it is marked "Needs More Info" and has been 15 days with no activity. To keep open, remove the stale label or comment on the issue, else this will be closed in 5 days.'
+        stale-issue-message: 'Help us move this issue forward. It has no activity after 15 days of requesting more information. A bot is marking the issue as stale. Please add additional information or this issued will be closed in 5 days.'
```

Comment:
I might be a bit clearer in this that a comment needs to be made on the issue as someone could, in theory, just edit their report with more information in a way that'll not show up as activity (unless I'm mistaken). 

I think the change is as simple as saying, "Please add additional information as a comment or this issued will be closed in 5 days."

Reply:
Updated, thanks!